chrysom/client.go

```diff
@@ -98,22 +98,14 @@ type ClientConfig struct {
 	// (Optional) If not provided, no auth headers are added.
 	Auth Auth
 
-	// MetricsProvider helps initialize metrics collectors.
-	// (Optional). By default a discard provider will be used.
-	MetricsProvider provider.Provider
-
 	// Logger to be used by the client.
 	// (Optional). By default a no op logger will be used.
 	Logger log.Logger
 
-	// Listener provides a mechanism to fetch a copy of all items within a bucket on
-	// an interval.
-	// (Optional). If not provided, listening won't be enabled for this client.
-	Listener Listener
-
-	// PullInterval is how often listeners should get updates.
-	// (Optional). Defaults to 5 seconds.
-	PullInterval time.Duration
+	// Listen contains data to configure
```

Comment:
I don't think you finished writing this comment?

Reply:
Good catch 👍